.env.example

```diff
@@ -1,8 +1,10 @@
-NODE_ENV=development
-PORT=4000
-JWT_SECRET=0a6b944d-d2fb-46fc-a85e-0295c986cd9f
-PG_DB=api
-PG_PORT=5432
-PG_HOST=localhost
-PG_USER=amida
-PG_PASSWD=''
\ No newline at end of file
+#NODE_ENV=development
+#PORT=4000
```

Comment:
I think we should probably add a prefix to the PORT and PG_* envvars a la @jmfadeley 's recent PR for all the microservices https://github.com/amida-tech/amida-auth-microservice/pull/48/files#diff-cbfd64a28982a1818f2b5f16e7f9d634R25

So they'd look something more like "APIBOILERPLATE_PG_HOST" rather than just "PG_HOST"